less/core/loading.less

```diff
@@ -1,26 +1,69 @@
-// Loading overlay styling
-// =======================
+@loading-anim-delay: 0.16s;
 
 .loading {
-    position: fixed!important; // Needs to be fixed otherwise loading layout can be incorrect
-    background-color: @transparency;
-    text-align: center;
-    
-    .no-opacity & {
-    	background-image: @transparency-fallback-png;
-    }
-
-    .loader-gif {
-        background-image: url(assets/ajax-loader.gif);
-        background-repeat: no-repeat;
-        background-position: center;
-        width: 128px;
-        height: 128px;
-        position: absolute;
-        top: 50%;
-        left: 50%;
-        margin-top: -64px;
-        margin-left: -64px;
-        .item-title;
-    }
-}
\ No newline at end of file
+  background-color: fade(@shadow, 50%);
+  color: @shadow-inverted;
+
+  &__inner,
+  &__inner:before,
+  &__inner:after {
+    width: 2rem;
+    height: 2rem;
+    border-radius: 50%;
+    -webkit-animation-fill-mode: both;
+    animation-fill-mode: both;
+    -webkit-animation: loading 1.25s infinite ease-in-out;
```

Comment:
Are we sure we need the prefixed version?

Reference: https://caniuse.com/#feat=css-animation.